splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeManager.java

```diff
@@ -27,4 +27,6 @@ default void endRollingUpgrade() throws StandardException {}
     default List<String> unloadRegions(String hostAndPort) throws StandardException {return null;}
 
     default List<String> loadRegions(String serverName) throws StandardException { return null;}
+
+    default void restartOlapServer() throws StandardException {}
```

Comment:
restartOlapServer() is implemented in [DB-10044](https://github.com/splicemachine/spliceengine-ee/pull/586/files#diff-bcaf7782ef4fbc67bdab5e571167ff48a0b9b624257571dba603d0166ca49a89R429-R440)

Reply:
Merged in restartOlapServer() with commit https://github.com/splicemachine/spliceengine/pull/5103/commits/0084db46bdb5a38d91dcb52fd9dda5749473d82f and updated function call in https://github.com/splicemachine/spliceengine/pull/5103/commits/30e1281c32782994f444bf3dc8ecc14d24e3c3df

Reply:
This is no longer need.

Reply:
Removed UpgradeManager.restartOlapServer() in https://github.com/splicemachine/spliceengine/pull/5103/commits/3a7193d118fa5a2ea41e5ef123668becee5044fa